embrace-android-sdk/embrace-proguard.cfg

```diff
@@ -1,7 +1,15 @@
 -keepattributes LineNumberTable, SourceFile
 
-## Keep classes used by hosted SDKs
+## Keep public API including enums
```

Comment:
After this, should we move all enums to the same package so we can just specify that instead and make sure future ones are also protected?